lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="exam-timer">
+<%- gettext("You're taking") %>
+<a href="<%- interpolate(gettext('%(exam_url_path)s'), { exam_url_path: exam_url_path }, true)%>" >
+
+<%- interpolate(gettext(' %(exam_display_name)s '), { exam_display_name: exam_display_name }, true) %>
+</a>
+<%- gettext(" exam as a proctored exam. Good Luck!") %>
```

Comment:
Suggest making some reference to being able to see the time remaining in the banner (replace "Good Luck!") but I'll be better able to come up with wording at a later stage when I can see the UI. 
Perhaps something like: "exam as a timed and proctored exam. A countdown timer will be visible (where?) that displays the amount of time remaining for you to complete the exam."
Thoughts? @chrisndodge or @pbaruah